rpm-tester/test_rpm.sh

```diff
@@ -9,20 +9,20 @@ npm install
 mkdir build
 ln -s /opt/ripple/bin/rippled build/rippled
 
-npm test
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/ripple/openssl/lib:/opt/ripple/boost/lib /opt/ripple/bin/rippled --unittest
 
 rc=$?; if [[ $rc != 0 ]]; then
-  echo "npm test failed"
+  echo "rippled --unittest failed"
   exit $rc
 fi
 
-/opt/ripple/bin/rippled --unittest
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/ripple/openssl/lib:/opt/ripple/boost/lib npm test
 
 rc=$?; if [[ $rc != 0 ]]; then
-  echo "rippled --unittest failed"
+  echo "npm test failed"
   exit $rc
 fi
 
 echo "tests passed"
 
-# make request to rippled-build-bot
\ No newline at end of file
+aws sqs send-message --queue-url https://sqs.us-west-2.amazonaws.com/356003847803/rippled-rpm-tested --message-body '{"results":"passed"}' --region us-west-2
```

Comment:
The tester doesn't currently have any info to provide.
I'd like to add the data passing in a separate PR.
Should I hold off on having the tester send a message to SQS until it has something meanful to submit?